internal/quickstart/messages.go

```diff
@@ -134,6 +135,17 @@ func sendFetchSDKInstructionsMsg(url string) tea.Cmd {
 	}
 }
 
+func sendReadSDKInstructionsMsg(filename string) tea.Cmd {
+	return func() tea.Msg {
+		content, err := os.ReadFile(fmt.Sprintf("internal/sdks/sdk_instructions/%s.md", filename))
+		if err != nil {
+			return errMsg{err: err}
+		}
+
+		return fetchedSDKInstructions{instructions: content}
```

Comment:
despite the name of the function we're still sending `fetchedSDKInstructions` message type - figured we can update either the message name or the function name when we stop fetching from the READMEs but this way we can still just handle this one message in the `showSDKInstructionsModel.Update` method.